addons/website_form/static/src/snippets/s_website_form/options.js

```diff
@@ -799,6 +808,18 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
         // We need to reload the existing type list.
         this.rerender = true;
     },
+    /**
+     * Rerenders the clone to avoid id duplicates.
+     * Rendering the list before replacing the field is needed
+     * for selects, in order to build this.listTable.
+     *
+     * @override
+     */
+    onClone() {
+        const field = this._getActiveField();
```

Comment:
There was a `_renderList` call here or something like that, right? Now that we use widgets, we magically do not need anything else to be done here? :open_mouth:

Reply:
this.listTable is now entirely managed by the we-list widget, created in _renderCustomXml for a new clone. I'll remove outdated comment